features/install-profiles/src/test/java/org/codice/ddf/features/profiles/test/ITInstallProfilesFeatures.java

```diff
@@ -52,7 +71,8 @@
         defaultLogging(),
         includeTestResources(),
         addFeatureRepo(FeatureUtilities.toFeatureRepo(FEATURE_REPO_PATH)),
-        addBootFeature(TestUtilitiesFeatures.testCommon()));
+        addBootFeature(TestUtilitiesFeatures.testCommon()),
+        addJavaxCache());
```

Comment:
❗️ We should not be updating the test to add the feature to the container. We should be updating the actual feature files to start this feature, if it's needed. Does installing all profiles with the unzipped DDF container manually work? If so, there's some mismatch between this test and the actual runtime installation.